src/components/docs-sidebar-nav-section.js

```diff
@@ -37,7 +37,7 @@ class DocsSidebarNavSectionContent extends React.Component {
     this.scrollToActiveTimeout = setTimeout(() => {
       // TODO
       // Investigate whether it’s possible to do this
-      // with a transitionend event instead of setTimeout
+      // with a transitioned event instead of setTimeout
```

Comment:
[`transitionend`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/transitionend_event) is the name of the event.